Library/ViewModels/PledgeAmountCellViewModelTests.swift

```diff
@@ -35,13 +35,61 @@ internal final class PledgeAmountCellViewModelTests: TestCase {
     self.vm.outputs.textFieldIsFirstResponder.observe(self.textFieldIsFirstResponder.observer)
   }
 
-  func testAmountAndCurrency() {
+  func testAmountCurrencyAndStepper_NoReward() {
```

Comment:
There is a lot of variation tests mostly because of the way `min` & `max` is calculated - for more details please see `minAndMaxPledgeAmount `